src/metabase/models/setting.clj

```diff
@@ -1054,7 +1054,15 @@
 
   Do you want to update something else when this setting changes? Takes a function which takes 2 arguments, `old`, and
   `new` and calls it with the old and new settings values. By default, the :on-change will be missing, and nothing
-  will happen, in [[call-on-change]] below."
+  will happen, in [[call-on-change]] below.
+
+  ###### `:feature`
+  If non-nil, determines the Enterprise feature flag required to use this setting. If the feature is not enabled,
+  the setting will behave the same as if `enabled?` returns `false` (see below).
+
+  ###### `enabled?`
+  Function which returns true if the setting should be enabled. If it returns false, the setting will throw an
+  exception when it is attempted to be set, and will return its default value when read. Defaults to always enabled."
```

Comment:
This was from my PR, right? Maybe you need to rebase against `feature-flags-integration`?

Reply:
Rebased and I think you missed it in your original PR

Reply:
Oh sorry you're right. I was confusing it with the comments in the `SettingDefinition` spec. Thanks for adding these